MessageTerminal/MessageTerminal.cxx

```diff
@@ -60,8 +65,12 @@ void move(istringstream& iss)
         else if (word == "neg") dir = DIR_NEGATIVE;
         else break;
 
-        if (comm.move(accel, hold_vel, dist, axis, dir)) cout << "OK" << endl;
-        else cout << "ERROR" << endl;
+        if (comm.move(accel, hold_vel, dist, axis, dir)) {
+            cout << "OK" << endl;
+        }
+        else {
+            printf("ERR: Received MSG ID: %d\n", comm.received_message().id);
```

Comment:
Use either `cout` or `printf` for consistency

Reply:
My "consistency" is currently `printf` for formatted output, `cout` for non-formatted. Is that consistent-ish-enough?

Reply:
Didn't realize this was the reasoning. Yes, that makes sense now thank you